### DISCUSSION TITLE
PATCH request gets 400 Bad Request when I play the stream using WHEP

### DISCUSSION THREAD

**Discussion by GPUbrainStorm** [bluenviron/mediamtx#3393] [Questions]
### Question

Hello guys,

I have set a WebRTC server using Mediamtx and it works normally but I have a problem with the PATCH request. the OPTIONS request works normally but:
when the player send a PATCH request to the server, it replies with 400 Bad Request: {"error":"sdp: invalid port value `RTP`"}

Here is the Request Payload:

a=ice-ufrag:rzTcHOPYHHoOntns
a=ice-pwd:wWhDhPIdhcdHCLqtJcuBhJHDkmSfVCHA
m=audio RTP/AVP 0
a=mid:0
a=candidate:669811394 1 udp 2122260223 192.168.1.130 61301 typ host generation 0 ufrag YaSk network-id 1
a=candidate:669811394 1 udp 2122260223 192.168.1.130 61302 typ host generation 0 ufrag YaSk network-id 1
a=candidate:2292605948 1 udp 1686052607 5.107.14.232 61301 typ srflx raddr 192.168.1.130 rport 61301 generation 0 ufrag YaSk network-id 1
a=candidate:2292605948 1 udp 1686052607 5.107.14.232 61302 typ srflx raddr 192.168.1.130 rport 61302 generation 0 ufrag YaSk network-id 1
a=candidate:1495490650 1 tcp 1518280447 192.168.1.130 9 typ host tcptype active generation 0 ufrag YaSk network-id 1
a=candidate:1495490650 1 tcp 1518280447 192.168.1.130 9 typ host tcptype active generation 0 ufrag YaSk network-id 1
a=candidate:4138549418 1 udp 41885951 51.178.47.208 50718 typ relay raddr 5.107.14.232 rport 61301 generation 0 ufrag YaSk network-id 1
a=candidate:4138549418 1 udp 41885951 51.178.47.208 49841 typ relay raddr 5.107.14.232 rport 61302 generation 0 ufrag YaSk network-id 1
a=candidate:1196236077 1 udp 41886207 146.59.44.10 63534 typ relay raddr 5.107.14.232 rport 61301 generation 0 ufrag YaSk network-id 1
a=candidate:1196236077 1 udp 41886207 146.59.44.10 50078 typ relay raddr 5.107.14.232 rport 61302 generation 0 ufrag YaSk network-id 1
a=candidate:2138367167 1 udp 8331263 51.195.101.185 53232 typ relay raddr 5.107.14.232 rport 50824 generation 0 ufrag YaSk network-id 1
a=candidate:2138367167 1 udp 8331263 51.195.101.185 50220 typ relay raddr 5.107.14.232 rport 50823 generation 0 ufrag YaSk network-id 1
a=end-of-candidates

and those are the headers:
:authority:
webrtc1.xxxx.com:8889
:method:
PATCH
:path:
/HoKSMCaster2/whep/43dcc0b2-a682-46f3-b1f3-9fc7ab4761ba
:scheme:
https
Accept:
*/*
Accept-Encoding:
gzip, deflate, br, zstd
Accept-Language:
en-US,en;q=0.9
Cache-Control:
max-age=0
Content-Length:
1669
Content-Type:
application/trickle-ice-sdpfrag
If-Match:
*
Origin:
https://vid.xxxx.com
Priority:
u=1, i
Referer:
https://vid.xxxx.com/
Sec-Ch-Ua:
"Google Chrome";v="125", "Chromium";v="125", "Not.A/Brand";v="24"
Sec-Ch-Ua-Mobile:
?0
Sec-Ch-Ua-Platform:
"Windows"
Sec-Fetch-Dest:
empty
Sec-Fetch-Mode:
cors
Sec-Fetch-Site:
same-site
User-Agent:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36

I couldnt understand what port is invalid, and what is a valid one anyway, Can anyone help with this problem?
Thank you,

**Comment by aler9** [comment]
Hello, i tried to replicate the issue but it's impossible without:
* server version
* server logs
* what client are you using (browser and whether you are using the bundled HTML page or a custom page)
* instructions on how to replicate

if you still have problems, open a bug report and provide details on how to replicate, as mentioned in the instructions.
